net/icmpv6/icmpv6_sendmsg.c

```diff
@@ -415,6 +415,8 @@ ssize_t icmpv6_sendmsg(FAR struct socket *psock, FAR struct msghdr *msg,
         {
           if (ret == -ETIMEDOUT)
             {
+              ret = -EAGAIN;
```

Comment:
Maybe can move it to `else` statement below?

Reply:
Done.